server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/cucumber/GetMessagesMethodStepdefs.java

```diff
@@ -252,35 +252,19 @@ public void appendMessageWithSpecialCase(String messageName, String mailbox) thr
 
     @Given("^the user has a message \"([^\"]*)\" in the \"([^\"]*)\" mailbox with flags \"([^\"]*)\"$")
     public void appendMessageWithFlags(String messageName, String mailbox, List<String> keywords) throws Exception {
-        Flags flags = getFlagsFromListOfKeywords(keywords);
-        appendMessage(messageName, flags);
-    }
-
-    private Flags getFlagsFromListOfKeywords(List<String> keywords) {
-        return new Keywords(keywords.stream()
-                .map(Keyword::new)
-                .collect(Guavate.toImmutableSet()))
+        Flags flags = Keywords.factory()
+            .fromList(keywords)
             .asFlags();
-    }
-
-    @Given("^the user has a message \"([^\"]*)\" in the \"([^\"]*)\" mailbox with Recent and Deleted flags \"([^\"]*)\"$")
-    public void appendMessageWithUnsupportedJmapFlags(String messageName, String mailbox, List<String> keywords) throws Exception {
-        Flags flags = getFlagsFromListOfKeywords(keywords);
-        appendMessage(messageName, flags);
-    }
-
-    @Given("^the user has a message \"([^\"]*)\" in the \"([^\"]*)\" mailbox with forwarded jmap flag \"([^\"]*)\"$")
-    public void appendMessageWithForwardedJmapFlags(String messageName, String mailbox, List<String> keywords) throws Exception {
-        Flags flags = getFlagsFromListOfKeywords(keywords);
         appendMessage(messageName, flags);
     }
 
     private void appendMessage(String messageName, Flags flags) throws Exception {
         ZonedDateTime dateTime = ZonedDateTime.parse("2014-10-30T14:12:00Z");
+        boolean isRecent = false;
```

Comment:
It had a scenario which append message with flags contain RECENT and DELETED 
And even we force isRecent=true, the tests are passed